pkg/unikontainers/unikontainers.go

```diff
@@ -355,7 +357,8 @@ func (u *Unikontainer) Delete() error {
 		return fmt.Errorf("cannot delete running unikernel: %s", u.State.ID)
 	}
 	unikernelType := u.State.Annotations[annotType]
-	unikernel, err := unikernels.New(unikernels.UnikernelType(unikernelType))
+	// There is no need to specify the version to execute a Delete operation
```

Comment:
Yes, so why do we need to set the version inside unikernels' constructor (New)?

Reply:
You are right, I changed it.